model/src/main/java/hu/blackbelt/judo/meta/jsl/validation/JslDslValidator.xtend

```diff
@@ -865,17 +865,17 @@ class JslDslValidator extends AbstractJslDslValidator {
     @Check
     def checkEnumLiteralMinimum(EnumDeclaration _enum) {
         if (_enum.literals.size < 1) {
-            error("Enumeration must have at least one member:"+_enum.name,
+            error("Enumeration must have at least one member: " + _enum.name + ".",
                 JsldslPackage::eINSTANCE.enumDeclaration_Literals,
                 ENUM_MEMBER_MISSING,
                 JsldslPackage::eINSTANCE.enumDeclaration.name)
         }
     }
 
     @Check
-    def checkEnumOridal(EnumLiteral literal) {
+    def checkEnumOrdinal(EnumLiteral literal) {
         if (literal.value > new BigInteger("9999")) {
-            error("Enumeration ordinal is greater than the maximum allowed 9999.",
+            error("Enumeration ordinal is greater than the maximum allowed 9999 at '" + literal.name + "'.",
```

Comment:
Magic number 9999. Can this be extracted as a static const or sth. like a parameter?